apps/frontend/src/api/inventory-api/dtos/inventory.relations.dto.ts

```diff
@@ -0,0 +1,6 @@
+import { InventoryDto } from "./inventory.dto";
+import { UserDto } from "../../user-api/dtos";
+
+export interface InventoryRelationsDto extends InventoryDto {
+	owner: UserDto;
+}
```

Comment:
Just for personal knowlege, what is the purpose of this DTO and why not have everything in the same DTO ?

Reply:
Given the `api.yaml` file, an `InventoryDTO` does **not** contain the owner.

However, @noahboegli gave previously an example (`/inventories?where[owner][name][$eq]=Paul`) in which it is possible to filter via a relation.

It could have been the same (and it actually is for the others DTO) if there's no extra properties for filtering.